pkg/configuration/config_manager/handler_util.go

```diff
@@ -193,9 +193,22 @@ func FilterSubPathVolumeMount(metas []ConfigSpecMeta, volumes []corev1.VolumeMou
 	var filtered []ConfigSpecMeta
 	for _, meta := range metas {
 		v := FindVolumeMount(volumes, meta.ConfigSpec.VolumeName)
-		if v == nil || v.SubPath == "" || meta.ReloadType == appsv1beta1.TPLScriptType {
+		if isSupportReloadAction(v, meta) {
```

Comment:
The name "FilterSubPathVolumeMount" seems is not a precise name, it not only filter out ConfigSpec with not-existed volumes, subpath, but also excludes any ConfigSpec of non-synchronous reload action.

Reply:
This comment is just a suggestion, has no relationship with the purpose of this pr.

Reply:
Suggest to split this function into two, one is FilterSubPathVolumeMount, the other with a better naming that explains the behavior expressed in "isSupportReloadAction".

Reply:
`
The name "FilterSubPathVolumeMount" seems is not a precise name, it not only filter out ConfigSpec with not-existed volumes, subpath, but also excludes any ConfigSpec of non-synchronous reload action.
`
Yes, when this function was written, the operator sync exec reload action was not supported, so just filter out configspecs using subpath.